tests/storage/conftest.py

```diff
@@ -1,7 +1,25 @@
+import pytest
+
+from tests.storage import install_randstream
+
+from typing import TYPE_CHECKING
+
+if TYPE_CHECKING:
+    from lib.vdi import VDI
+    from lib.vm import VM
```

Comment:
Using `TYPE_CHECKING` here seems to go in the opposite direction from the "add some type annotations" commit (on which I suggested to move the removal of those into a separate commit)